src/ol/View.js

```diff
@@ -1104,30 +1132,74 @@ class View extends BaseObject {
   }
 
   /**
-   * Rotate the view around a given coordinate.
-   * @param {number} rotation New rotation value for the view.
+   * Adds relative coordinates to the center of the view. Any extent constraint will apply.
+   * @param {import("./coordinate.js").Coordinate} deltaCoordinates Relative value to add.
+   * @api
+   */
+  move(deltaCoordinates) {
```

Comment:
As already said, I think `adjustCenter` would be consistent with the other new methods.